build.gradle.kts

```diff
@@ -100,14 +87,11 @@ tasks {
     }
 
     register("runServer", JavaExec::class) {
-        environment("LEGACY_API_URL", "http://localhost:8090/person/dittnav-legacy-api")
-        environment("EVENT_HANDLER_URL", "http://localhost:8092")
-        environment("CORS_ALLOWED_ORIGINS", "localhost:9002")
-
-        environment("OIDC_ISSUER", "http://localhost:9000")
-        environment("OIDC_DISCOVERY_URL", "http://localhost:9000/.well-known/openid-configuration")
-        environment("OIDC_ACCEPTED_AUDIENCE", "stubOidcClient")
-        environment("OIDC_CLAIM_CONTAINING_THE_IDENTITY", "pid")
+        println("Setting default environment variables for running with DittNAV docker-compose")
+        DockerComposeDefaults.environomentVariables.forEach { (name, value) ->
+            println("Setting the environment variable $name")
+            environment(name, value)
```

Comment:
For å kunne kalle metoden environment ser det ut til at vi må lage oss et eget gradle-plugin. Det kan vi jo se på på sikt.